### ISSUE TITLE
Error: Unable to get creeper blast radius

### ISSUE THREAD

**Issue by Nethya** [Funwayguy/Epic-Siege-Mod#94]
Hello, I've setup ESM and been running into this problem constantly, the server continues running but this error pops up every couple of minutes.

```
[Server thread/ERROR] [epicsiegemod]: Unable to get creeper blast radius
java.lang.IllegalAccessException: Class funwayguy.epicsiegemod.ai.utils.CreeperHooks can not access a member of class net.minecraft.entity.monster.EntityCreeper with modifiers "private"
        at sun.reflect.Reflection.ensureMemberAccess(Unknown Source) ~[?:1.8.0_171]
        at java.lang.reflect.AccessibleObject.slowCheckMemberAccess(Unknown Source) ~[?:1.8.0_171]
        at java.lang.reflect.AccessibleObject.checkAccess(Unknown Source) ~[?:1.8.0_171]
        at java.lang.reflect.Field.getInt(Unknown Source) ~[?:1.8.0_171]
        at funwayguy.epicsiegemod.ai.utils.CreeperHooks.getExplosionSize(CreeperHooks.java:41) [CreeperHooks.class:?]
        at funwayguy.epicsiegemod.ai.ESM_EntityAICreeperSwell.func_75250_a(ESM_EntityAICreeperSwell.java:38) [ESM_EntityAICreeperSwell.class:?]
        at net.minecraft.entity.ai.EntityAITasks.func_75774_a(SourceFile:94) [xf.class:?]
        at net.minecraft.entity.EntityLiving.func_70626_be(EntityLiving.java:763) [vq.class:?]
        at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:2350) [vp.class:?]
        at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:577) [vq.class:?]
        at net.minecraft.entity.monster.EntityMob.func_70636_d(EntityMob.java:45) [ade.class:?]
        at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:2170) [vp.class:?]
        at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:295) [vq.class:?]
        at net.minecraft.entity.monster.EntityMob.func_70071_h_(EntityMob.java:50) [ade.class:?]
        at net.minecraft.entity.monster.EntityCreeper.func_70071_h_(EntityCreeper.java:172) [acs.class:?]
        at net.minecraft.entity.Entity.func_70098_U(Entity.java:2013) [vg.class:?]
        at net.minecraft.entity.EntityLivingBase.func_70098_U(EntityLivingBase.java:2480) [vp.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:1985) [amu.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832) [oo.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:1952) [amu.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:2050) [amu.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832) [oo.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:1952) [amu.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:1756) [amu.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613) [oo.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:396) [nz.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_171]
[15:26:52] [Server thread/ERROR] [epicsiegemod]: Unable to get creeper blast radius 
```
List of mods: [https://i.imgur.com/pItLNVS.png](https://i.imgur.com/pItLNVS.png)

**Comment by Nethya** [comment]
Bug happens in EpicSiegeMod-13.161.jar
No bug in EpicSiegeMod-13.160.jar

**Comment by Funwayguy** [comment]
Fixed in 13.162
